supervisor/bootstrap.py

```diff
@@ -274,14 +274,14 @@ def supervisor_debugger(coresys: CoreSys) -> None:
     if not coresys.config.debug:
         return
     # pylint: disable=import-outside-toplevel
-    import ptvsd
+    import debugpy
```

Comment:
You don't have to import this outside of the top-level anymore.

debugpy can safely be imported top level.